ParquetCSVImporter/Import Program.cs

```diff
@@ -0,0 +1,94 @@
+using System.Text;
+using System.IO;
+using System.Collections.Generic;
+using System.Globalization;
+using CsvHelper;
+using CsvHelper.TypeConversion;
+using ParquetClassLibrary.Sandbox.ID;
+using ParquetClassLibrary.Sandbox.Parquets;
+using ParquetCSVImporter.ClassMaps;
+
+namespace ParquetCSVImporter
+{
+    /// <summary>
+    /// A program that reads in Parquet definitions from CSV files, and outputs them as JSON.
+    /// TODO Add ability to output them as JSON!
+    /// </summary>
+    class MainClass
+    {
+        /// <summary>The location of the Designer files.</summary>
+        public static readonly string SearchPath =
+#if DEBUG
+            Directory.GetParent(System.IO.Directory.GetCurrentDirectory()).Parent.Parent.FullName;
```

Comment:
comment: oooooh, that's how you can solve this. I've run in to this issue before with the new Visual Studio project files. Thanks for that.

Reply:
Yay!  :D